x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_create_rules/route.ts

```diff
@@ -88,6 +94,18 @@ export const bulkCreateRulesRoute = (
                 });
               }
             }
+            const isExceptionListValid = await isValidExceptionList({
```

Comment:
Can we add a unit test case for this check for each of `bulk_create_rules/route.test.ts`, `bulk_patch_rules/route.test.ts`, ... etc?